### PR TITLE
Enable respective protobuf test version on confluent

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**

### CODE REVIEW

File: test/test_verify.py [hunk 1]
```diff
@@ -341,11 +342,11 @@ def runTestSet(driver, testSet, nameSalt, pressure):
                      testAvrosrAvrosr, testNativeStringAvrosr, testNativeStringJsonWithoutSchema,
                      testNativeComplexSmt, testNativeStringProtobuf, testConfluentProtobufProtobuf]
 
-    # Adding StringJsonProxy test at the end
+    # clean is set to true for all tests
     testCleanEnableList1 = [True, True, True, True, True, True, True, True, True, True, True]
     testSuitEnableList1 = []
     if testSet == "confluent":
-        testSuitEnableList1 = [True, True, True, True, True, True, True, True, True, True, False]
+        testSuitEnableList1 = [True, True, True, True, True, True, True, True, True, False, True]
```
Comment (sfc-gh-japatel) [inline test/test_verify.py:349]: I might be wrong here but can we install blueapron protobuf on confluent? (Seems like we can, just confirming)

If yes, I should set second to last value to true too. (corresponding to testNativeStringProtobuf)
CC: @sfc-gh-zli

Comment (sfc-gh-zli) [reply test/test_verify.py]: Seems like tests are failing?

Comment (sfc-gh-japatel) [reply test/test_verify.py]: Mostly because of a rebalance.

**Comment by sfc-gh-japatel** [comment]
CC: @sfc-gh-rramachandran / @sfc-gh-anwang

**Pull request closed without merging**